Docker/validate_sc2.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+"""Validate SC2."""
+
+import argparse
+import json
+
+import pandas as pd
+
+
+def main(submission, entity_type, results="results.json"):
+    """Validate submission and write results to JSON.
+
+    Args:
+        submission: input file
+        entity: Synapse entity type
+        results: output file
+    """
+
+    invalid_reasons = []
+
+    if not submission.endswith(".csv"):
+        invalid_reasons.append(
+            "Prediction file must be a two-column CSV file.")
+    else:
+        predictions = pd.read_csv(submission, sep=",")
+
+        header = list(predictions.columns)
+        try:
+            assert header == ["lab_id", "responder"]
```

Comment:
This is one way to do it.  A simplier way would be just to do

```
if header != ....:
    invalid_reasons.append(...)
```

Reply:
Ah, so switch my exception handling; got it!